src/EditorFeatures/VisualBasicTest/MakeFieldReadonly/MakeFieldReadonlyTests.vb

```diff
@@ -0,0 +1,532 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.MakeFieldReadonly
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.MakeFieldReadonly
+    Public Class MakeFieldReadonlyTests
+        Inherits AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest
+
+        Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace) As (DiagnosticAnalyzer, CodeFixProvider)
+            Return (New VisualBasicMakeFieldReadonlyDiagnosticAnalyzer(),
+                New VisualBasicMakeFieldReadonlyCodeFixProvider())
+        End Function
+
+        <Theory, Trait(Traits.Feature, Traits.Features.CodeActionsMakeFieldReadonly)>
```

Comment:
i haven't looked deeply at this file.  But i wanted to check if you have tests for all the interesting vb field forms.  i.e. they can do things like:

```vb
    Dim x As Integer, y As String
```

or

```vb
Dim x, y As Integer, z, w As String
```

or:

```vb
    Dim x, y() As Integer, z, w As String
```
(note the parens after y).

Reply:
➡️ Added in commit 38db528